### DISCUSSION TITLE
Getting previous BuildId only from Generate Release Notes plugin

### DISCUSSION THREAD

**Discussion by khopithan** [rfennell/AzurePipelines#1326] [Q&A] [answered]
Hi @rfennell,

I used your plugin and it is working perfectly. But I need to edit some output from this plugin. 

* **Previous Build**: {{compareBuildDetails.buildNumber}}

Output - Previous Build: devops_test.19340

But what I need is only - Previous Build: 19340

How can I remove the branch name from the previous buildid? or is there any predefine variable for this?

**Comment by rfennell** [comment, marked as answer]
The build na e that includes the branch and the version is just a string, so you can use any of the handlebars extensions I ship in the task.

If these don't do what you want, you can always write a custom extension in javascript. 

In both cases check the rep wiki for details

**Reply by khopithan** [reply]
Thank you. And I did this by implementing Library variables.
